src/main/java/AttendanceTracker/App.java

```diff
@@ -3,7 +3,8 @@
  */
 package AttendanceTracker;
 
-import static AttendanceTracker.UserChooseYear.userChooseYearStyle;
+
+import static AttendanceTracker.YearOptions.userChooseYearStyle;
```

Comment:
you don't need to import, we can make static references to the YearOptions methods